src/application/tasks/ventilation.c

```diff
@@ -23,11 +24,11 @@
  */
 static int start_current_breath_cycle = 0;
 
-/*
- * the time the current breath cycle was supposed to start
- */
-static int breath_cycle_duration = 125; //to do replace this value by the real one
-
+uint32_t calculate_breath_cycle_duration(){
+  uint32_t respiratory_rate = 15; // todo: Retrieve clinician respiratory rate

```

Comment:
you can directly implement your todo

however we can create another issue to do this specific code so the choice is yours either implement the todo or create an issue